includes/class-wc-ecfb-front-end.php

```diff
@@ -236,6 +242,7 @@ public function checkout_billing_fields( $fields ) {
 		if ( isset( $fields['billing_postcode'] ) ) {
 			$new_fields['billing_postcode'] = $fields['billing_postcode'];
 			$new_fields['billing_postcode']['class'] = array( 'form-row-last', 'address-field' );
+			$new_fields['billing_postcode']['type'] = 'tel';
```

Comment:
Dava para aproveitar o script ai e trocar o tipo de campo do CEP, de `text` para `tel` se for no Brasil, acho que não seria um problema em mobile.